testing/engine/variables.go

```diff
@@ -171,9 +171,9 @@ var _ = profile.RegisterProfile(profile.Profile{
 	Rules: `
 SecRequestBodyAccess On
 SecRule ARGS:key "@contains sensitive" "id:9,phase:2,pass"
-SecRule ARGS:key[value] "@contains sensitive" "id:10,phase:2,pass,log,logdata:'Message from rule 10: %{MATCHED_VAR_NAME} %{MATCHED_VAR}'"
-SecRule ARGS:key2[] "@contains newValue" "id:11,phase:2,pass,setvar:'tx.macro_exp_var=%{ARGS.key2[]}',chain"
+SecRule ARGS_POST:key[value] "@contains sensitive" "id:10,phase:2,pass,log,logdata:'Message from rule 10: %{MATCHED_VAR_NAME} %{MATCHED_VAR}'"
+SecRule ARGS:key2[] "@contains newValue" "id:11,phase:2,pass,setvar:'tx.macro_exp_var=%{ARGS_POST.key2[]}',chain"
 	SecRule TX:macro_exp_var "@contains newValue"
-SecRule ARGS:key2[name] "@contains PaYlOaD" "id:12,phase:2,pass,log,logdata:'Message from rule 12: %{MATCHED_VAR_NAME}, macro expansion: %{ARGS.key2[name]}'"
+SecRule ARGS_POST:key2[name] "@contains PaYlOaD" "id:12,phase:2,pass,log,logdata:'Message from rule 12: %{MATCHED_VAR_NAME}, macro expansion: %{ARGS_POST.key2[name]}'"
```

Comment:
Thanks! We should have dedicated tests (to be checked) with multiphase checking the ARGS -> ARGS_POST conversion, so this test should be good like this.